.github/workflows/integration-tests-on-release.yaml

```diff
@@ -6,6 +6,9 @@ on:
     branches:
       - master
       - release-*
+    paths-ignore:
```

Comment:
Let's actually revert this one. While it's not strictly necessary, it's nice to have the release tests run more often, even if not affected by the docs.

Reply:
It's still showing three commits, did you force push the new commit to the branch?

Reply:
Actually, it looks like you picked up a couple extra commits from the history. Try `git rebase upstream/master`, then force push again

Reply:
Yes, it is even on the GitHub Actions run, I don´t know why is it showing commits [abb5a12](https://github.com/rook/rook/pull/12291/commits/abb5a12afd9eeafc00d273b75115abecf9ce44a2) and [94b533c](https://github.com/rook/rook/pull/12291/commits/94b533cb3ba0d970b89b87696cad21e4fba5842c) since my git rebase command only got my own commits. For some reason they show up as a part of the pull request

![image](https://github.com/rook/rook/assets/97261428/fb197d2c-72e4-4564-aebe-ce778053cce4)

Reply:
If you click on [1222fe7](https://github.com/rook/rook/pull/12291/commits/1222fe7275647ebb3bdea989fab37ea5143ef173) it even has the combination of my 3 commits message's in there (confirming the successfull squash)

Reply:
Did you see [this comment](https://github.com/rook/rook/pull/12291#discussion_r1210576993)?

Reply:
Now it should be good to go! I'm sorry for some Git issues I've commited on this run, still not in an expert level of the tool..

Hope this solves the issue! Thanks for letting me contribute to the project!